docs/usage/TYPE_SYNC.md

```diff
@@ -72,7 +72,7 @@ __Note__ The statistics object contains the processing time of the last batch on
  
 #### More examples of how to use the sync
  
- 1. [Sync from another CTP project as a source](https://github.com/commercetools/commercetools-sync-java/tree/master/src/integration-test/java/com/commercetools/sync/integration/ctpprojectsource/types/TypeSyncIT.java).
+ 1. [Sync from another CTP project as a source](https://github.com/commercetools/commercetools-sync-java/tree/master/src/integration-test/java/com/commercetools/sync/integration/ctpprojectsource/type/TypeSyncIT.java).
```

Comment:
Good catch! but I think the actual package name needs to be changed in this case ;)

Reply:
to be consistent with the others

Reply:
renamed:  https://github.com/commercetools/commercetools-sync-java/pull/361/commits/42b01682ba7e17871d17901bafbbdad66653a686